website/src/pages/intro.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import {Redirect} from '@docusaurus/router';
+
+export default function Home() {
+    return <Redirect to="/" />;
```

Comment:
@Fashander I removed index.js as there is nothing fancy about it. If later we need it, we will restore it from history, but based on the content of index.js, I don't think we will reuse it even if we migrate the main website to docusaurus.

Reply:
That's okay, thanks. We can always do that when we need it later. 👍🏾